rubocop.test.yml

```diff
@@ -21,3 +21,7 @@ Naming/VariableNumber:
   - 'test/**/*_test.rb' # minitest
   - 'spec/**/*_spec.rb' # RSpec
 
+Metrics/ClassLength:
+  Exclude:
+    - 'test/**/*_test.rb' # minitest
+    - 'spec/**/*_spec.rb' # RSpec
```

Comment:
Nit picking, but in RSpec we don't define tests with class. Could you remove this one?